assets/js/googlesitekit/datastore/user/date-range.js

```diff
@@ -112,16 +146,20 @@ export const selectors = {
 	 * @param {Object}  [options]               Options parameter. Default is: {}.
 	 * @param {boolean} [options.compare]       Set to true if date ranges to compare should be included. Default is: false.
 	 * @param {number}  [options.offsetDays]    Number of days to offset. Default is: 0.
-	 * @param {string}  [options.referenceDate] Used for testing to set a static date. Default is: `new Date()` (today/now).
+	 * @param {string}  [options.referenceDate] Used for testing to set a static date. Default is the datastore's reference date.
 	 * @param {boolean} [options.weekDayAlign]  Set to true if the compared date range should be aligned for the weekdays. Default is: false.
 	 * @return {DateRangeReturnObj}             Object containing dates for date ranges.
 	 */
 	getDateRangeDates( state, {
 		compare = false,
 		offsetDays = 0,
-		referenceDate = getDateString( new Date() ),
+		referenceDate,
 		weekDayAlign = false,
 	} = {} ) {
+		if ( ! referenceDate ) {
+			referenceDate = state.referenceDate;
+		}
+
```

Comment:
I'm fairly certain this can be simplified like so:
```suggestion
		referenceDate = state.referenceDate,
		weekDayAlign = false,
	} = {} ) {
```
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters#Earlier_parameters_are_available_to_later_default_parameters